src/svg/ds6/icons.svg

```diff
@@ -3,4 +3,7 @@
     <symbol id="icon-long-arrow-right" viewBox="0 0 32 32">
         <path d="M20.997 6.333l-1.546 1.546 7.236 7.236h-26.686v2.313h26.516l-7.25 7.264 1.546 1.546 10.045-10.045-9.86-9.86z"></path>
     </symbol>
+    <symbol id="icon-close" viewBox="0 0 17 17">
```

Comment:
Should SVG IDs follow the "camelCase for IDs" principle?

@ianmcburnie, thoughts?

Reply:
I am aware by asking this it would apply to the other icons as well, I'm just asking if we want to stick with what is usually done in the markup to make sure IDs and class names don't ever "mix".

Reply:
@seangates I've never heard of using "camelCase" for svg ids, where / why is this recommended?

Reply:
We've never used camelCase for IDs. SVGs in DS4 all used a dash separator.

Reply:
Okay. I wasn't sure if there could be a distinction given it's an ID, and not a class. I mean, that's always how I've done IDs in markup, whether SVG or otherwise. (BTW, I'm _not_ and SVG expert.)